modules/slugs/includes/class-qtranslate-slug.php

```diff
@@ -1091,44 +1095,17 @@ public function hide_slug_box() {
     }
 
     /**
-     * Hide auttomatically the wordpress slug box in edit posts page.
-     * User should still be able to use it if needed.
-     */
-    public function remove_defaultslug_meta_box() {
-
-        if ( is_admin() ) {
-            if ( ! current_user_can( 'manage_options' ) ) {
-                remove_meta_box( 'slugdiv', 'post', 'normal' );
-            }
-        }
-    }
-
-    /**
-     * Creates a metabox for every post, page and post type available.
+     * Creates a metabox for every post type available.
      */
     public function add_slug_meta_box() {
-
-        if ( function_exists( 'add_meta_box' ) ) {
-
-            $context  = apply_filters( "qts_admin_meta_box_context", "side" );
-            $priority = apply_filters( "qts_admin_meta_box_priority", "high" );
-
-            add_meta_box( 'qts_sectionid', __( 'Slugs per language', 'qtranslate' ), array(
-                &$this,
-                'draw_meta_box'
-            ), 'post', $context, $priority );
-            add_meta_box( 'qts_sectionid', __( 'Slugs per language', 'qtranslate' ), array(
-                &$this,
-                'draw_meta_box'
-            ), 'page', $context, $priority );
-
-            foreach ( get_post_types( array( '_builtin' => false ) ) as $ptype ) {
-                add_meta_box( 'qts_sectionid', __( 'Slugs per language', 'qtranslate' ), array(
-                    &$this,
-                    'draw_meta_box'
-                ), $ptype, $context, $priority );
-            }
-        }
+        $context  = apply_filters( "qts_admin_meta_box_context", "side" );
+        $priority = apply_filters( "qts_admin_meta_box_priority", "high" );
```

Comment:
I guess these two filters are not useful anymore.

Reply:
agreed, removed